src/action.js

```diff
@@ -11,13 +11,26 @@ export async function run() {
   *** ACTION RUN - START ***
   `)
   const MAX_MESSAGE_LENGTH = 280
+  const MAX_MEDIA_ELEMENT = 4
   const message = core.getInput('message', { required: true })
   const appKey = core.getInput('twitter-app-key', { required: true })
   const appSecret = core.getInput('twitter-app-secret', { required: true })
   const accessToken = core.getInput('twitter-access-token', { required: true })
   const accessSecret = core.getInput('twitter-access-token-secret', {
     required: true
   })
+  const media = core
+    .getInput('media', { required: false })
+    ?.split('\n')
```

Comment:
I don't think these conditional statements are needed, because getInput should always return a string, see https://github.com/actions/toolkit/tree/main/packages/core#inputsoutputs

Reply:
Yes, conditional removed